app/src/main/res/values/strings.xml

```diff
@@ -495,4 +495,6 @@ Otherwise, you can download another keyboard in the app store. Popular keyboards
     <string name="quest_fee_only_at_hours">Yes, but only…</string>
     <string name="quest_fee_not_at_hours">Yes, but not…</string>
     <string name="quest_fee_hours_title">at the following times:</string>
+    <string name="quest_atm_deposit_title">Can you deposit cash in this ATM?</string>
+    <string name="quest_atm_deposit_title_operator">"Can you deposit cash in %s ATM?"</string>
```

Comment:
That sound a little bit strange to me...

Reply:
The operator question sounds a little bit weird...
"Can you deposit cash in this ATM of %s" sounds better IMO...

Reply:
I agree

Reply:
I can change that

Reply:
Done